### ISSUE TITLE
How to satisfy SA1629 when the last word is a URL

### ISSUE THREAD

**Issue by AArnott** [DotNetAnalyzers/StyleCopAnalyzers#2831]
I have comments such as this:

```cs
    /// <remarks>
    /// See the spec for JSON-RPC over HTTP here: https://www.jsonrpc.org/historical/json-rpc-over-http.html
    /// </remarks>
```

SA1629 tells me that I should end my comment with a period. But a period at the end of a URL will be treated as part of the URL, which makes my URL incorrect. I can't always think of a reasonable word to end a sentence with to artificially satisfy this rule.

Should the rule permit comments that end with a URL?

**Comment by AArnott** [comment]
Ah. I didn't know you could use `href` here. :) Thanks.
